svm/tests/conformance.rs

```diff
@@ -245,8 +245,8 @@ fn run_fixture(fixture: InstrFixture, filename: OsString, execute_as_instr: bool
 
     mock_bank.override_feature_set(feature_set);
     let batch_processor = TransactionBatchProcessor::<MockForkGraph>::new(42, 2, HashSet::new());
+    batch_processor.set_fork_graph_in_program_cache(Arc::new(RwLock::new(MockForkGraph {})));
 
-    let fork_graph = Arc::new(RwLock::new(MockForkGraph {}));
     {
         let mut program_cache = batch_processor.program_cache.write().unwrap();
```

Comment:
Consecutive write locks.

Reply:
Yeah, this kinda stinks. Do you suggest closing this PR then?

Reply:
Perhaps this is better? https://github.com/anza-xyz/agave/pull/3203

Reply:
> Perhaps this is better? #3203

Yes, it looks better!